### PR TITLE
[tests-only][full-ci] use separate web config file for keycloak pipeline

### PR DESCRIPTION
**Pull request by S-Panta**
- separate web config file for keycloak-ocis setup
- removed unnecessary steps/methods/codes

### CODE REVIEW

**Review by saw-jan — APPROVED** [review]

**Review by nabim777 — APPROVED** [review]

**Comment by saw-jan** [comment]
Could be related to https://github.com/owncloud/ocis/issues/8669

**Comment by saw-jan** [comment]
> Could be related to [owncloud/ocis#8669](https://github.com/owncloud/ocis/issues/8669)

Might be NO.
@S-Panta Could you try updating the version (to same in https://github.com/owncloud/ocis/pull/8687)

**Comment by saw-jan** [comment]
ocis server crashed with keycloak

**Comment by saw-jan** [comment]
I see problem with web config file. I will have a look

**Comment by saw-jan** [comment]
> I see problem with web config file. I will have a look

https://github.com/owncloud/ocis/issues/8703

File: .drone.star [hunk 1]
```diff
@@ -677,11 +673,8 @@ def e2eTests(ctx):
         depends_on = []
         steps = skipIfUnchanged(ctx, "e2e-tests") + \
                 restoreBuildArtifactCache(ctx, "pnpm", ".pnpm-store") + \
-                restoreBuildArtifactCache(ctx, "playwright", ".playwright") + \
                 installPnpm() + \
-                installPlaywright() + \
```
Comment (saw-jan) [inline .drone.star:682]: don't need to install browser from playwright. the image used in test run step has the required chrome browser

**Review by ScharfViktor — APPROVED** [review]

**Pull request merged**